access reviews to v1 - using doc stub generator (#12207)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/accessreviewscope.md

---
title: "accessReviewScope resource type"

> description: "In the Azure AD access reviews feature, the accessReviewScope represents what entities will be reviewed in an access review."

Reviewer: ```suggestion
description: "Represents the entities that need to be reviewed in an access review."
```

author: "isabelleatmsft"
localization_priority: Normal
ms.prod: "governance"
doc_type: resourcePageType
---

# accessReviewScope resource type

Namespace: microsoft.graph

> The **accessReviewScope** defines what entities will be reviewed in an [accessReviewScheduleDefinition](accessreviewscheduledefinition.md). It is an abstract type that is inherited by [accessReviewQueryScope](accessreviewqueryscope.md), [principalResourceMembershipsScope](principalresourcemembershipsscope.md) and [accessReviewReviewerScope](accessreviewreviewerscope.md). 

Reviewer: ```suggestion
The **accessReviewScope** defines entities that need to be reviewed in an [accessReviewScheduleDefinition](accessreviewscheduledefinition.md). It is an abstract type that is inherited by [accessReviewQueryScope](accessreviewqueryscope.md), [principalResourceMembershipsScope](principalresourcemembershipsscope.md) and [accessReviewReviewerScope](accessreviewreviewerscope.md). 
```
